Tests/test_imagegrab.py

```diff
@@ -15,8 +15,9 @@ class TestImageGrab:
         sys.platform not in ("win32", "darwin"), reason="requires Windows or macOS"
     )
     def test_grab(self):
-        if os.environ.get("USERNAME") == "ContainerAdministrator":
+        if os.environ.get("USERNAME") == "ContainerAdministrator":  # pragma: no cover
```

Comment:
Or can we use a `@pytest.mark.skipif` decorator as well?

Reply:
Oh right, yeah.